.golangci.yml

```diff
@@ -63,12 +64,26 @@ linters-settings:
     forbid:
       - '\brequire\.New\b(# Use package-level functions with explicit TestingT)?'
       - '\bassert\.New\b(# Use package-level functions with explicit TestingT)?'
-      - '[^\/]\net\/rpc\b(# Use consul-net-rpc/net/rpc for net/rpc needs)?'
-      - '\bgithub\.com\/hashicorp\/net-rpc-msgpackrpc\b(# Use consul-net-rpc/net-rpc-msgpackrpc)?'
-      - '\bgithub\.com\/hashicorp\/go-msgpack\b(# Use consul-net-rpc/go-msgpackrpc)?'
     # Exclude godoc examples from forbidigo checks.
     # Default: true
     exclude_godoc_examples: false
+  gomodguard:
+    blocked:
+      # List of blocked modules.
+      modules:
+        # Blocked module.
+        - net/rpc:
```

Comment:
This is stdlib and won't appear in a go.mod unfortunately. You'd have to block it another way

Reply:
Actually, does it work? I guess it might still work depending on how naively it checks things

Reply:
```
package main

import (
	netrpc "net/rpc"

	codec "github.com/hashicorp/go-msgpack/codec"
	msgpackrpc "github.com/hashicorp/net-rpc-msgpackrpc"
	// "github.com/stretchr/testify/assert"
	// "github.com/stretchr/testify/require"
)

func gomodguard() int {

	_ = msgpackrpc.CallWithCodec
	_ = netrpc.Accept
	_ = codec.NewEncoder

	return 0
}

// func forbidigo() {
// 	_ = require.New
// 	_ = assert.New
// }
```

```
$ make lint
--> Running go golangci-lint
canary.go:6:2: import of package `github.com/hashicorp/go-msgpack/codec` is blocked because the module is in the blocked modules list. `github.com/hashicorp/consul-net-rpc/go-msgpack` is a recommended module. (gomodguard)
	codec "github.com/hashicorp/go-msgpack/codec"
	^
canary.go:7:2: import of package `github.com/hashicorp/net-rpc-msgpackrpc` is blocked because the module is in the blocked modules list. `github.com/hashicorp/consul-net-rpc/net-rpc-msgpackrpc` is a recommended module. (gomodguard)
	msgpackrpc "github.com/hashicorp/net-rpc-msgpackrpc"
```